### DISCUSSION TITLE
Make "change race" service be able to be gifted to a friend.

### DISCUSSION THREAD

**Discussion by Anwyyn** [chromiecraft/chromiecraft#7115] [Suggestions]
Right now "change race" service allows a player to change any character from they own accoun to a different race from the same faction. But i was wondering if this service can actually be gifted to a friend. Think this isn't a game changer feature to noone since it literally can be purchasable by anyone, but what can't be done yet is to gift it to a friend.

**Comment by Nyeriah** [comment]
Hello,

the issue with gifting services is that the other party has no choice of turning down the gift if they turn up to be unwanted

**Reply by Anwyyn** [reply]
Ty for the reply!

If so, wouldnt be an interesting idea that the other party gets a "code" via mail box which can be redeemed in cc website? Just a suggestion tho, i dont know if what i am asking is too complicated or even already have been analyzed in the past
